test/DynamicExpresso.UnitTest/LiteralsTest.cs

```diff
@@ -75,6 +77,219 @@ public void Numeric_Literals()
 			Assert.AreEqual(-.022M, target.Eval("-.022M"));
 		}
 
```

Comment:
I think it should be useful to add the following test:

```
		[Test]
		public void Numeric_Literals_DefaultTypes()
		{
			var target = new Interpreter();
			Assert.IsInstanceOf(typeof(System.Int32), target.Eval("81"));
			Assert.IsInstanceOf(typeof(System.Double), target.Eval("81.5"));
			Assert.IsInstanceOf(typeof(System.Int64), target.Eval("23423423423434"));
		}
```

To verify that by default numbers are interpreted as pre documentation.

Reply:
I have added the Numeric_Literals_DefaultTypes test.